### PR TITLE
fix: add donlouie

### PR DESCRIPTION
**Pull request by donlouie**
## 
[x] - add profile donlouie.json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/742"></a>

### CODE REVIEW

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
Looks good to me, however, this file needs formatting...

It would be great if it can be changed to -
```json
{
  "name": "Louie",
  "bio": "Web Developer, Exploring Open Source",
  "avatar": "https://github.com/donlouie.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/donlouie",
      "icon": "github"
    }
  ]
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

File: public/data/donlouie.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "avatar": "https://github.com/donlouie.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/donlouie",
+      "icon": "github"
+    }
+  ]
+}
+
```
Comment (Panquesito7) [inline public/data/donlouie.json:13]:
```suggestion
```

**Review by donlouie — COMMENTED** [review]
-

**Review by Panquesito7 — APPROVED** [review]

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**